Fixed Gitlab event collector fetch (#30379)
demisto/content: Packs/GitLab/ReleaseNotes/2_2_19.md

#### Integrations

##### GitLab Event Collector

> - Fixed an issue where the ***fetch-events*** failed in case there are a large number of events and it didn't had enough time to fetch everything before it raised on timeout.

Reviewer: ```suggestion
- Fixed an issue where the ***fetch-events*** command failed on timeout.
```
